phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,62 @@
-						<!-- ELSE -->
-							<a href="{messagerow.U_VIEW_PM}" class="topictitle">{messagerow.SUBJECT}</a>
-						<!-- ENDIF -->
-						<!-- IF messagerow.S_AUTHOR_DELETED -->
-							<br /><em class="small">{L_PM_FROM_REMOVED_AUTHOR}</em>
-						<!-- ENDIF -->
-						<!-- IF messagerow.S_PM_REPORTED -->
-							<a href="{messagerow.U_MCP_REPORT}">
-								<i class="icon fa-exclamation fa-fw icon-red" aria-hidden="true"></i><span class="sr-only">{PM_REPORTED}</span>
-							</a>
-						<!-- ENDIF --> <!-- IF messagerow.ATTACH_ICON_IMG --><i class="icon fa-paperclip fa-fw" aria-hidden="true"></i> <!-- ENDIF --><br />
-						<!-- IF S_SHOW_RECIPIENTS -->{L_MESSAGE_TO} {messagerow.RECIPIENTS}<!-- ELSE -->{L_MESSAGE_BY_AUTHOR} {messagerow.MESSAGE_AUTHOR_FULL} &raquo; {messagerow.SENT_TIME}<!-- ENDIF -->
-
-						</div>
-					</dt>
-					<!-- IF S_SHOW_RECIPIENTS --><dd class="info"><span>{L_SENT_AT}{L_COLON} {messagerow.SENT_TIME}</span></dd><!-- ENDIF -->
-					<!-- IF S_UNREAD --><dd class="info"><!-- IF messagerow.FOLDER --><a href="{messagerow.U_FOLDER}">{messagerow.FOLDER}</a><!-- ELSE -->{L_UNKNOWN_FOLDER}<!-- ENDIF --></dd><!-- ENDIF -->
-					<dd class="mark"><input type="checkbox" name="marked_msg_id[]" value="{messagerow.MESSAGE_ID}" /></dd>
-				</dl>
-			</li>
-		<!-- END messagerow -->
-
-		</ul>
-	<!-- ELSE -->
-		<p><strong>
-			<!-- IF S_COMPOSE_PM_VIEW and S_NO_AUTH_SEND_MESSAGE -->
-				<!-- IF S_USER_NEW -->{L_USER_NEW_PERMISSION_DISALLOWED}<!-- ELSE -->{L_NO_AUTH_SEND_MESSAGE}<!-- ENDIF -->
-			<!-- ELSE -->
-				{L_NO_MESSAGES}
-			<!-- ENDIF -->
-		</strong></p>
-	<!-- ENDIF -->
-
-	<!-- IF FOLDER_CUR_MESSAGES neq 0 -->
-		<fieldset class="display-actions">
-			<div class="left-box"><label for="export_option">{L_EXPORT_FOLDER}{L_COLON} <select name="export_option" id="export_option"><option value="CSV">{L_EXPORT_AS_CSV}</option><option value="CSV_EXCEL">{L_EXPORT_AS_CSV_EXCEL}</option><option value="XML">{L_EXPORT_AS_XML}</option></select></label> <input class="button1 button button-form-bold" type="submit" name="submit_export" value="{L_GO}" /><br /></div>
-			<select name="mark_option">{S_MARK_OPTIONS}{S_MOVE_MARKED_OPTIONS}</select> <input class="button1 button button-form-bold" type="submit" name="submit_mark" value="{L_GO}" />
-			<div><a href="#" onclick="marklist('viewfolder', 'marked_msg', true); return false;">{L_MARK_ALL}</a> &bull; <a href="#" onclick="marklist('viewfolder', 'marked_msg', false); return false;">{L_UNMARK_ALL}</a></div>
-		</fieldset>
-
-		<hr />
-
-		<div class="action-bar bottom">
-			<!-- INCLUDE display_options.html -->
-			<input type="hidden" name="cur_folder_id" value="{CUR_FOLDER_ID}" />
-
-			<div class="pagination">
-				{TOTAL_MESSAGES}
-				<!-- IF .pagination -->
-					<!-- INCLUDE pagination.html -->
-				<!-- ELSE -->
-					 &bull; {PAGE_NUMBER}
-				<!-- ENDIF -->
-			</div>
-		</div>
-	<!-- ENDIF -->
+	</div>
 
+	<div class="pm-action-bar-bottom">
```

Comment:
Since you didn't touch options and pagination, it should be displayed as it was. So horizontal line above it, font smaller.